src/util/fieldHelpers.js

```diff
@@ -171,3 +172,25 @@ export const pickCustomFieldProps = (
       : pickedElements;
   }, []);
 };
+
+/**
+ * Checks if the currency is compatible based on process type and supported currencies.
+ *
+ * @param {string} processName - The name of the process being checked.
+ * @param {Array<string>} supportedCurrencies - List of currencies supported by the marketplace.
+ * @returns {boolean} - True if the currency is compatible, false otherwise.
+ */
+export const checkIfCompatibleCurrency = (
```

Comment:
Change the name to something more Stripe specific. (Some customizers will introduce some local payment processor anyway.)

Reply:
Those parameter/function names are just suggestions.